libraries/lib-files/FileNames.cpp

```diff
@@ -626,7 +626,12 @@ void FileNames::AddUniquePathToPathList(const FilePath &pathArg,
                                           FilePaths &pathList)
 {
    wxFileNameWrapper pathNorm { pathArg };
-   pathNorm.Normalize();
+   // https://github.com/audacity/audacity/issues/6448 :
+   // Do not seek to expand environment variables here: it is not needed, and
+   // wxWidgets has an issue doing so - See
+   // https://github.com/wxWidgets/wxWidgets/issues/19214
+   const auto flags = wxPATH_NORM_ALL & ~wxPATH_NORM_ENV_VARS;
```

Comment:
One thing that bothers me a bit is that `AddUniquePathToPathList` isn't unique anymore

Reply:
You mean it was unique before this change and now it isn't ? I don't understand.

Reply:
No, I just didn't think well enough about it, please ignore that comment